### PR TITLE
test(react-color-picker): added tests for ColorSlider

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
- Added tests for ColorSlider
- Fixed defaultValue

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/ColorSlider.test.tsx [hunk 1]
```diff
@@ -34,4 +34,105 @@ describe('ColorSlider', () => {
       </div>
     `);
   });
+
+  it('renders horizontal ColorSlider correctly', () => {
+    const { container } = render(<ColorSlider defaultValue={50} min={0} max={100} />);
+    expect(container).toMatchInlineSnapshot(`
```
Comment (Hotell) [inline packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/ColorSlider.test.tsx:40]: wondering how can one determine that this test covers that something renders horizonally/vertically...

after further investigation it seems only "differentiator" is css variable `--fui-Slider--direction: 180deg`, if that's correct can we rather not use snapshots and test only what makes sense ?

Comment (ValentinaKozlova) [reply packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/ColorSlider.test.tsx:40]: Yeah, I'll remove it. We'll test vertical/horizontal in VR tests

Comment (Hotell) [reply packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/ColorSlider.test.tsx]: It's ok to test it here if needed, what I was saying is to not use snapshots if not necessary rather scoped approach like these other spec you wrote.

**Review by Hotell — APPROVED** [review]

**Pull request merged**